__tests__/buildTranslationFiles.spec.ts

```diff
@@ -9,35 +9,78 @@ import { resolveProjectBasePath } from '../src/utils/resolve-project-base-path';
 
 const sourceRoot = '__tests__';
 
-const m = 'missing';
-
-function gKeys(len: number, prefix?: string) {
-  let expected = {};
-  for (let i = 1; i <= len; i++) {
-    expected[prefix ? `${prefix}.${i}` : i] = m;
-  }
-
-  return expected;
+const defaultValue = 'missing';
+
+/** Returns an array from start to end inclusive, by increments of step. */
+const getRange = (start: number, end: number, step = 1): number[] => {
```

Comment:
Please use a simple for loop just like `gKeys` originally used, no need for complexity, this is a simple generator.
You can add the `start` property just like you did, but I think this should also be an options object where you have default values that match like today:

```ts
function generateKeys({start = 1, len, prefix})
```